.env-production

```diff
@@ -6,6 +6,7 @@ API_NAMESPACE=v2
 API_DATA_NAMESPACE=Database
 API_ADMIN_NAMESPACE=Admin
 API_ERRORS_ENDPOINT=/error-events
+AUTH_BASE_URL=
```

Comment:
@lhorne-gavant just making sure you're aware of this new env var we'll want to add to our build scripts